src/EditorFeatures/Core/Shared/Extensions/SymbolDisplayPartExtensions.cs

```diff
@@ -27,14 +28,14 @@ public static Run ToRun(this SymbolDisplayPart part, IClassificationFormatMap fo
             return run;
         }
 
-        public static TextBlock ToTextBlock(this ImmutableArray<SymbolDisplayPart> parts, ClassificationTypeMap typeMap)
+        public static TextBlock ToTextBlock(this ImmutableArray<SymbolDisplayPart> parts, ClassificationTypeMap typeMap, TextWrapping textWrapping = TextWrapping.NoWrap)
```

Comment:
Indeed the only other caller sets the resultant TextBlock to use TextWrapping.Wrap. I can probably remove the parameter always wrap the TextBlock.